src/Pact/Native/Ops.hs

```diff
@@ -194,7 +202,12 @@ sqrtDef = defRNative "sqrt" sqrt' unopTy ["(sqrt 25)"] "Square root of X."
   sqrt' fi as@[TLiteral a _] = do
     unlessExecutionFlagSet FlagDisablePact43 $
       when (not (litGt0 a)) $ evalError' fi "Sqrt must be non-negative"
-    (unopd sqrt) fi as
+#if defined(ghcjs_HOST_OS)
+    unopd sqrt fi as
+#else
+    decimalSqrt <- ifExecutionFlagSet' FlagDisableNewPow (unopd sqrt) (unopd trans_sqrt)
```

Comment:
Should `FlagDisableNewPow` renamed into something like `FlagDisableNewTranscendentals`

Reply:
Went with `FlagDisableNewTrans`. Sound good?